scripts/init.sh

```diff
@@ -24,10 +24,34 @@ for var in "${vars_to_check[@]}"; do
   export "${var}"
 done
 
+# Setting the Node Role
+env_file_exist "${ROOT_DIR}/${ENV_FILE}"
+# shellcheck disable=SC1090
+source "${ROOT_DIR}/${ENV_FILE}" || { echo "Error: could not source ${ROOT_DIR}/${ENV_FILE} file. Fix it before proceeding any further.  Exiting..."; exit 1; }
+
+if [ -z "${SCNODE_ROLE:-}" ]; then
+  read -rp "Which kind of node would you like to start ? Choose between 'rpc' or 'forger': " SCNODE_ROLE_VALUE
+  SCNODE_ROLE="${SCNODE_ROLE_VALUE}"
+  while [[ ! "${SCNODE_ROLE_VALUE}" =~ ^(rpc|forger)$  ]]; do
+    echo -e "Error: The options have to be 'rpc' or 'forger'. Try again...\n"
+    read -rp "Which kind of node would you like to start ? Choose between 'rpc' or 'forger': " SCNODE_ROLE_VALUE
+    SCNODE_ROLE="${SCNODE_ROLE_VALUE}"
+  done
+  sed -i'' "s/SCNODE_ROLE=/SCNODE_ROLE=${SCNODE_ROLE}/g" ${ROOT_DIR}/${ENV_FILE}
+  if [ ${SCNODE_ROLE} = "forger" ]; then
+    sed -i'' 's/SCNODE_FORGER_ENABLED=/SCNODE_FORGER_ENABLED=true/g' ${ROOT_DIR}/${ENV_FILE}
+    compose_file=docker-compose-forger.yml
+  else
+    sed -i'' 's/SCNODE_FORGER_ENABLED=/SCNODE_FORGER_ENABLED=false/g' ${ROOT_DIR}/${ENV_FILE}
+    compose_file=docker-compose-simple.yml
```

Comment:
Why are you assigning compose_file here if later you are also running `select_compose_file` function? :thinking: